set.mm

```diff
@@ -205833,6 +205865,27 @@ C_ dom ( S sSet <. I , E >. ) ) $=
       IUJABLBABCGDEFMABNOUIPZUJQIJZBUKCQIUKCDARTQEDARSUAUBUCULQBUDBUEUFUGUH $.
   $}
 
+  ${
+    ressbasssg.r $e |- R = ( W |`s A ) $.
+    ressbasssg.b $e |- B = ( Base ` W ) $.
+    $( The base set of a restriction to ` A ` is a subset of ` A ` and the base
+       set ` B ` of the original structure.  (Contributed by SN,
+       10-Jan-2025.) $)
+    ressbasssg $p |- ( Base ` R ) C_ ( A i^i B ) $=
```

Comment:
could be used to shorten ressbasss

Reply:
Done! I've credited you for the shortening.